src/app/clusters/scenes/scenes.cpp

```diff
@@ -186,13 +177,43 @@ void emAfPluginScenesServerPrintInfo()
             emberAfCoreFlush();
 #endif // ZCL_USING_COLOR_CONTROL_CLUSTER_SERVER
 #ifdef ZCL_USING_DOOR_LOCK_CLUSTER_SERVER
-            emberAfCorePrint(" door %x", entry.lockStateValue);
+            if (entry.lockStateValue.IsNull())
+            {
+                emberAfCorePrint(" door null");
+            }
+            else
+            {
+                emberAfCorePrint(" door %x", to_underlying(entry.lockStateValue.Value()));
+            }
 #endif
 #ifdef ZCL_USING_WINDOW_COVERING_CLUSTER_SERVER
-            emberAfCorePrint(" Window percentage Lift %3u, Tilt %3u", entry.currentPositionLiftPercentageValue,
-                             entry.currentPositionTiltPercentageValue);
```

Comment:
this was going to explode at certain point ..... gosh

Reply:
There's a lot of that in scenes.  :(